styles.css

```diff
@@ -243,3 +244,50 @@ figure:hover figcaption {
   height: 575px;
   background-image: url(images/about.png);
 }
+
+/* NEWSLETTER section */
+
+.newsletter {
+  margin-top: 357px;
+  display: flex;
+  justify-content: center;
+}
+
+.subscription {
+  width: 1240px;
+  height: 395px;
+  padding: 70px 100px;
+  background-color: #293241;
+  color: white;
+  border-radius: 20px;
+  background-image: url(images/tourists.png);
+  background-position: right;
+  background-repeat: no-repeat;
+  
+}
+
+.subscription h1 {
+  margin-bottom: 10px;
+}
+
+.subscription p {
+  width: 505px;
+  height: 53px;
+  margin-top: 9px;
+  margin-bottom: 54px;
+}
+
+.email {
+  width: 358px;
+  height: 52px;
+  border-radius: 26px;
+  font-size: 18px;
+  padding-left: 30px;
+  margin-right: 15px;
+  border: none;
+}
+
+form {
```

Comment:
geriau sukuri klasę, o ne naudoti elemento selectorių. Taip geriau, nes jeigu vėliau reikėtų įterpti dar vieną formą, tai jai jau atomatiškai galiotų šitas stilius. bet gal to nereikia? Ypač jeigu projektas pasidaro ilgas, daug kodo, tada sunk atsekti iš kur atsiranda tie stiliai.